ops/docker/op-stack-go/Dockerfile

```diff
@@ -47,14 +47,11 @@ ARG TARGETARCH
 # The "id" defaults to the value of "target", the cache will thus be reused during this build.
 # "sharing" defaults to "shared", the cache will thus be available to other concurrent docker builds.
 
-# For now fetch the v1 cannon binary from the op-challenger image
-#FROM --platform=$BUILDPLATFORM us-docker.pkg.dev/oplabs-tools-artifacts/images/op-challenger:v1.1.0 AS cannon-builder-0
+FROM --platform=$BUILDPLATFORM us-docker.pkg.dev/oplabs-tools-artifacts/images/cannon:v1.0.0-alpha.3 AS cannon-builder-0
```

Comment:
Should we do the `cannon:v1.0.0` final release prior to merging this or stick to alpha.3 and then update after we've seen this PR merge onto develop and not get reverted for a little while?

Reply:
Let's stick with alpha.3